src/component/Calculator.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable react/no-unused-state */
+/* eslint-disable no-mixed-spaces-and-tabs */
+/* eslint-disable no-tabs */
+import React from 'react';
+import calculate from '../logic/calculate';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class MyCalculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      operation: null,
+      next: null,
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+	handleChange = (event) => {
+	  const buttonName = event.target.innerHTML;
+	  const results = calculate(this.state, buttonName);
+	  this.setState(results);
+	}
+
+	render() {
+	  const { total, operation, next } = this.state;
+	  return (
+  <section>
+    <div className="container">
+      <table>
+        <thead>
+          <tr>
+            <td
+              colSpan="4"
+              className="inputs"
+              onClick={this.handleChange}
+              role="presentation"
+            >
+              {total}
+              {operation}
+              {next}
+            </td>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td onClick={this.handleChange} role="presentation">AC</td>
+            <td onClick={this.handleChange} role="presentation">+</td>
+            <td onClick={this.handleChange} role="presentation">%</td>
+            <td className="orange">÷</td>
```

Comment:
* I can see no event was added to this button here as a result the divide button doesn't work 
[scrnli_11_30_2022_6-45-38 AM.webm](https://user-images.githubusercontent.com/52497246/204716779-c73df1e7-0aee-4ab7-b3bb-fb709093eae5.webm)

Reply:
Thanks @Davidosky007  for pointing it out, I must have forgotten it. 
Let me fix it.